src/main/webapp/app/exercises/text/shared/manual-text-selection/manual-text-selection.component.ts

```diff
@@ -20,10 +21,14 @@ export class ManualTextSelectionComponent {
     public hostRectangle: SelectionRectangle | undefined;
     public selectedText: string | undefined;
 
-    constructor(public textAssessmentAnalytics: TextAssessmentAnalytics, protected route: ActivatedRoute) {
+    constructor(public textAssessmentAnalytics: TextAssessmentAnalytics, protected route: ActivatedRoute, private translateService: TranslateService) {
         textAssessmentAnalytics.setComponentRoute(route);
     }
 
+    get assessText(): string {
+        return this.translateService.instant('artemisApp.textAssessment.editor.assess');
+    }
+
```

Comment:
Shouldn't this be rather an attribute (Coding Guidelines no function calls in template code)?

Reply:
@Olimbia23